### PR TITLE
Address regular audit of broken external wiki links

### LINKED ISSUE #3962: Regular audit of external wiki links
### Description
I re-ran the python script used from the [previous audit](https://github.com/w3f/polkadot-wiki/pull/3442) as a regular check and discovered a few new 400 and 500 level errors outlined below.  There were many more 300 level redirect status codes but quickly clicking through a few, they appear to navigate to valid pages after the redirect.  A more thorough look would be required to confirm all of them.

It may be worth spending some time trying to write a similar script for internal links to assist with tasks such as https://github.com/w3f/polkadot-wiki/issues/3875.  Docusaurus provides a list when building the project but it doesn't seem to be exhaustive as it currently only provides one result:
```
Exhaustive list of all broken links found:

- On source page path = /docs/learn-staking:
   -> linking to learn-cross-consensus.md (resolved as: /docs/learn-cross-consensus.md)
```

### 400 and 500 Status Codes
|File|Error|Tag|Url|
|---|---|---|---|
|[build-smart-contracts.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-smart-contracts.md)|**404**|`Substrate ink! Workshop`|https://docs.substrate.io/tutorials/smart-contracts/develop-contract/|
|[build-node-management.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-node-management.md)|**404**|`monitor node metrics tutorial`|https://docs.substrate.io/tutorials/get-started/node-metrics/|
|[build-tools-index.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-tools-index.md)|**404**|`Tutorials`|https://dotleap.com/tag/tutorial/|
|[build-parachains.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-parachains.md)|**404**|`Parachain tutorials`|https://docs.substrate.io/tutorials/connect-other-chains/relay-chain/|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Golang Chainsafe`|https://github.com/ChainSafe/gossamer/tree/development/lib/scale|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Haskell`|https://github.com/airalab/hs-web3/tree/master/src/Codec|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Polkadothub Indexer`|https://github.com/figment-networks/polkadothub-indexer|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Parallel Finance`|https://github.com/parallel-finance/parallel/blob/feature-oracle/pallets/ocw-oracle/src/lib.rs|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Apollo`|https://github.com/ComposableFi/composable/tree/main/frame/oracle|
|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Societal`|https://github.com/sctllabs/societal_node|
|[claims.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/claims.md)|**404**|`Can I claim multiple Ethereum addresses to one single Polkadot account?`|https://support.polkadot.network/support/solutions/articles/65000068824-can-i-claim-multiple-ethereum-addresses-to-one-single-polkadot-account-|
|[kusama-community.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/kusama/kusama-community.md)|**404**|`Polkadot's Latest Research`|https://research.web3.foundation/en/latest/polkadot/|
|[kusama-community.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/kusama/kusama-community.md)|**404**|`Dotleap.com Tutorials`|https://dotleap.com|
|[maintain-endpoints.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/maintain/maintain-endpoints.md)|**404**|`Radium Block`|https://radiumblock.io|
|[learn-phragmen.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/learn/learn-phragmen.md)|**500**|`"Phragmén’s Voting Methods and Justified Representation"`|https://aaai.org/ocs/index.php/AAAI/AAAI17/paper/download/14757/13791|
|[learn-phragmen.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/learn/learn-phragmen.md)|**500**|`Phragmén’s Voting Methods and Justified Representation`|https://aaai.org/ocs/index.php/AAAI/AAAI17/paper/download/14757/13791|

### PR DESCRIPTION
**Pull request by alfarok**
Addresses #3962

Fixed = ✔️
Removed = 🗑️ (verified content was deleted or no longer available) 
Timing Out = ⌛ (potentially still active)
No Action = ❌

|Fixed|File|Error|Tag|Url|
|---|---|---|---|---|
|✔️|[build-smart-contracts.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-smart-contracts.md)|**404**|`Substrate ink! Workshop`|https://docs.substrate.io/tutorials/smart-contracts/develop-contract/|
|✔️|[build-node-management.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-node-management.md)|**404**|`monitor node metrics tutorial`|https://docs.substrate.io/tutorials/get-started/node-metrics/|
|🗑️|[build-tools-index.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-tools-index.md)|**404**|`Tutorials`|https://dotleap.com/tag/tutorial/|
|✔️|[build-parachains.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-parachains.md)|**404**|`Parachain tutorials`|https://docs.substrate.io/tutorials/connect-other-chains/relay-chain/|
|✔️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Golang Chainsafe`|https://github.com/ChainSafe/gossamer/tree/development/lib/scale|
|✔️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Haskell`|https://github.com/airalab/hs-web3/tree/master/src/Codec|
|🗑️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Polkadothub Indexer`|https://github.com/figment-networks/polkadothub-indexer|
|🗑️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Parallel Finance`|https://github.com/parallel-finance/parallel/blob/feature-oracle/pallets/ocw-oracle/src/lib.rs|
|✔️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Apollo`|https://github.com/ComposableFi/composable/tree/main/frame/oracle|
|✔️|[build-open-source.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/build/build-open-source.md)|**404**|`Societal`|https://github.com/sctllabs/societal_node|
|✔️|[claims.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/claims.md)|**404**|`Can I claim multiple Ethereum addresses to one single Polkadot account?`|https://support.polkadot.network/support/solutions/articles/65000068824-can-i-claim-multiple-ethereum-addresses-to-one-single-polkadot-account-|
|✔️|[kusama-community.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/kusama/kusama-community.md)|**404**|`Polkadot's Latest Research`|https://research.web3.foundation/en/latest/polkadot/|
|🗑️|[kusama-community.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/general/kusama/kusama-community.md)|**404**|`Dotleap.com Tutorials`|https://dotleap.com|
|🗑️|[maintain-endpoints.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/maintain/maintain-endpoints.md)|**404**|`Radium Block`|https://radiumblock.io|
|✔️|[learn-phragmen.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/learn/learn-phragmen.md)|**500**|`"Phragmén’s Voting Methods and Justified Representation"`|https://aaai.org/ocs/index.php/AAAI/AAAI17/paper/download/14757/13791|
|✔️|[learn-phragmen.md](https://github.com/w3f/polkadot-wiki/blob/master/main/docs/learn/learn-phragmen.md)|**500**|`Phragmén’s Voting Methods and Justified Representation`|https://aaai.org/ocs/index.php/AAAI/AAAI17/paper/download/14757/13791|

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -8,4 +8,5 @@
 !*/
 
 # Ignore translated docs (not currently maintained)
-/polkadot-wiki/translated_docs/
\ No newline at end of file
+/polkadot-wiki/translated_docs/
+/docs/build/build-open-source.md
```
Comment (alfarok) [inline .prettierignore:12]: This prevents prettier from trying to format `build-open-source.md`.

File: docs/build/build-parachains.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ With {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, you will ge
 of the box if you build your application as a parachain. The [XCM format](../learn/learn-xcm.md)
 allows any parachains to communicate by passing messages between them. Furthermore, as bridges to
 other chains are connected (such as those to Bitcoin or Ethereum)
-{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} parachains will be able to
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s parachains will be able to
 communicate with these as well.
```
Comment (alfarok) [inline docs/build/build-parachains.md:91]: Interestingly these `'` in conditional rendering brackets cause the syntax highlighting in vscode work incorrectly but **does not impact rendering**.  This is an easy fix for a simple case.  I haven't encountered this in other areas of the docs yet.

File: docs/build/build-smart-contracts.md [hunk 3]
```diff
@@ -198,11 +199,19 @@ Try deploying a smart contract to Moonbeam by following their [documentation][mo
 
 #### Astar
 
-[Astar Network](https://astar.network/) supports the building of dApps with EVM and WASM smart contracts and offers developers true interoperability. True interoperability with cross-consensus messaging [XCM](https://wiki.polkadot.network/docs/learn-xcm) and cross-virtual machine [XVM](https://github.com/AstarNetwork/). We are made by developers and for developers. Astar’s unique Build2Earn model empowers developers to get paid through a dApp staking mechanism for the code they write and dApps they build.
+[Astar Network](https://astar.network/) supports the building of dApps with EVM and WASM smart
```
Comment (alfarok) [inline docs/build/build-smart-contracts.md:202]: I think unformatted (not pretty) code still sneaks in to the repo from time to time when changes are made directly via the GitHub UI.

File: docs/build/build-parachains.md (hunk 2 above)
Comment (profradha) [reply docs/build/build-parachains.md:91]: Will keep this in mind while reviewing the PRs. Thanks!

File: docs/build/build-smart-contracts.md (hunk 3 above)
Comment (profradha) [reply docs/build/build-smart-contracts.md:202]: Got it! Thanks!

File: docs/general/kusama/kusama-community.md [hunk 4]
```diff
@@ -21,8 +21,7 @@ anyone doing so is likely trying to scam you.**
   Information on hosting meetups, applying for funding, and materials for running it.
 - [Support Knowledgebase](https://support.polkadot.network/support/home) and
   [Polkadot Support Contact](https://support.polkadot.network).
-- [Polkadot's Latest Research](https://research.web3.foundation/en/latest/polkadot/) - also applies
-  to Kusama.
+- [Polkadot's Latest Research](https://web3.foundation/research/) - also applies to Kusama.
```
Comment (profradha) [inline docs/general/kusama/kusama-community.md]:
```suggestion
- [Polkadot's Latest Research](https://research.web3.foundation/en/latest/polkadot/overview.html) - also applies to Kusama.
```

File: docs/maintain/maintain-endpoints.md [hunk 5]
```diff
@@ -71,7 +71,6 @@ service, and additional metrics.
 - [OnFinality](https://onfinality.io)
 - [Dwellir](https://dwellir.com)
 - [Pinknode](https://pinknode.io)
-- [Radium Block](https://radiumblock.io)
```
Comment (profradha) [inline docs/maintain/maintain-endpoints.md:74]: https://radiumblock.com/ - please update with this URL

**Review by DrW3RK — CHANGES_REQUESTED** [review]
I just checked in the Support channel for that claims related doc. Let's wait for their response and then merge this PR.

**Review by filippoweb3 — APPROVED** [review]
Hey thanks Keith for this audit. Everything looks good to me after addressing what is currently requested by Radha. I will keep in mind to avoid ' within the conditional rendering.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**